pydicom/dataset.py

```diff
@@ -1387,9 +1402,16 @@ def decompress(self):
 
         - ``Dataset.file_meta.TransferSyntaxUID`` is updated to non-compressed
           form
-        - :func:`~pydicom.dataelem.DataElement.is_undefined_length`
+        - :attr:`~pydicom.dataelem.DataElement.is_undefined_length`
           is ``False`` for the (7fe0,0010) *Pixel Data* element.
 
+        Parameters
+        ----------
+        handler: module or None
+            The pixel handler module that shall be used to decode the data.
```

Comment:
I was wondering if it would be much nicer (avoiding ugly import statements to get the handler to pass here) if we let people specify handlers as a string, e.g. "gdcm", "numpy", etc.  Could have a dict in config that then converts these to names to the function.  Or perhaps we could just append "_handler" to the passed name and look up the correct functions that way.

Reply:
I have been thinking along the same lines - I will change that in this case.

Reply:
Ok, done now.